csrc/utils.h

```diff
@@ -31,7 +31,9 @@ bool is_cpu_scalar(const c10::TensorType& tensor_type);
 
 // TODO: merge these two
 // check if input is compatible with 32b index mode
-int8_t getCommonDeviceCUDA(const at::ArrayRef<c10::IValue>& inputs);
+int8_t getCommonDeviceCUDA(
```

Comment:
Not your change, but the comment doesn't seem to make sense as the index type is not resolved here.